### PR TITLE
Update de_DE.lang

### PR DESCRIPTION
**Pull request by ReneMuetti**
Update to 0.11.2.01

### CODE REVIEW

**Comment by XCompWiz** [comment]
This seems to break the "myst.symbol" keys, changing them to "mystsymbol" without the dot. That won't work. Please fix this and flatten the commit again. :)

**Comment by ReneMuetti** [comment]
I do not quite understand what is meant. I've been using as a template the current en_US.lang. I aheb leave all key names and adjusted only the translation strings. what exactly is "wrong"?

**Comment by XCompWiz** [comment]
"myst.symbol" has been replaced with "mystsymbol" in your file.  You lost the "." part in all of them.  Probably a text replace.  Look at the diff near the bottom and you'll see it.

File: lang/de_DE.lang [hunk 1]
```diff
@@ -58,93 +70,102 @@ achievement.myst.symbol.desc=Finde ein Symbol
...
-myst.symbol.WeatherNorm.name=Normales Wetter
-myst.symbol.WeatherOff.name=Kein Wetter
-myst.symbol.WeatherOn.name=Ewiges Wetter
-myst.symbol.WeatherRain.name=Ewiger Regen
-myst.symbol.WeatherSlow.name=Langsames Wetter
-myst.symbol.WeatherSnow.name=Ewiger Schnee
-myst.symbol.WeatherStorm.name=Ewiger Sturm
-myst.symbol.Villages.name=Dörfer
-myst.symbol.Void.name=Leere Welt
+mystsymbol.BioConGrid.name=Gittermäßige Biomverteilung
```
Comment (XCompWiz) [inline lang/de_DE.lang]: See? Here.
The previous key was 
myst.symbol.BioConHuge.name
now the key is
mystsymbol.BioConHuge.name

You changed the key by accident.

**Comment by TheJulianJES** [comment]
There are grammar problemes in there, many problems, like "das" should be "dass" and there are missing commas.

**Comment by ReneMuetti** [comment]
I've created a translation that extends the very narrow version but here the current part. It is, of course, in the interests of all that help many people. I am not perfect and translations unterleigen a continuous further processing. If there are errors, it can correct any problöemlos this course.

**Comment by XCompWiz** [comment]
This commit looks like it might not be going anywhere. Not only does it have technical issues, it apparently has linguistic issues as well.  I ask these issues either be fixed or the commit closed.
Sorry, and thank you.

**Pull request closed without merging**